Process_WOA2018.ipynb

```diff
@@ -4,43 +4,38 @@
    "cell_type": "markdown",
```

Comment:
<p>do we want to keep the original notebook using seawater and add a new notebook using gsw?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/raphaeldussin/remap_WOA18/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
@raphaeldussin now that I have been using `GSW` (py, fortran), I have a hard time to work with `seawater`: the diff in args and function names confuses me. I would suggest using GSW.

Reply:
> do we want to keep the original notebook using seawater and add a new notebook using gsw?
> 
> 
> _Reply via [ReviewNB](https://app.reviewnb.com/raphaeldussin/remap_WOA18/pull/1/discussion/#comment-1107901468)_

✅ via https://github.com/sanAkel/remap_WOA18/commit/03206a9019190fc6f57f1837eb856ad1a0dc71c4